src/test/java/io/xstefank/wildfly/bot/PRTitleCheckTest.java

```diff
@@ -61,6 +61,47 @@ void incorrectTitleCheckFailTest() throws IOException {
             });
     }
 
+    @Test
+    void incorrectTitleCheckFailFormatConfigMissingTest() throws IOException {
```

Comment:
I'm missing positive test cases, aka `correctTitleCheckSuccessFormatConfigMissingTest` and `correctTitleCheckSuccessFormatConfigNullTest` when `gitHubJson = GitHubJson.builder(VALID_PR_TEMPLATE_JSON).build()`. Given that they should pass of course.